packages/cli/templates/react/src/components/Counter.tsx

```diff
@@ -1,20 +1,26 @@
-import { FunctionComponent } from 'react'
-import { SetValue } from '@storyblok/field-plugin'
-import Button from './button/Button'
+import { PluginActions, PluginState } from '@storyblok/field-plugin'
 
-type CounterFunc = FunctionComponent<{
-  setValue: SetValue
-  value: unknown
-}>
-const Counter: CounterFunc = ({ setValue, value }) => {
-  const label = typeof value !== 'number' ? 0 : JSON.stringify(value)
+type Props = {
+  setValue: PluginActions['setValue']
+  data: PluginState
```

Comment:
`Counter` only uses `data.value`, soi f we that instead of `data`, `Counter` can be agnostic of `@storyblok/field-plugin`

Reply:
Yes, but I'd prefer keeping this as-is, because it shows `data: PluginState` is an object that contains `value` property, which is set by `setValue()`. I think the current version shows the current data structure of field-plugin.